### PR TITLE
Provide stateless function support

### PR DESCRIPTION
**Pull request by troutowicz**
Support for React's stateless components.

cc @ericelliott

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ interface reactDesc {
   ...methods?: Function
 }
 
-stamp.compose(...desc?: stamp|reactDesc|specDesc[]): stamp
+stamp.compose(...objs?: Function|reactDesc|specDesc[]): stamp
```
Comment (ericelliott) [inline README.md]: Don't we also want to support stamps?

```js
stamp.compose(...args?: Function|stamp|reactDesc|specDesc[]): stamp
```

Something like...

```js
isStamp = typeof obj === 'function' && typeof obj.compose === 'function';
isPureComponent = !isStamp && typeof obj === 'function';
```

Comment (troutowicz) [inline README.md]: The way I thought of this was that a `stamp` is a special `Function`.

**Comment by troutowicz** [comment]
I think this PR is overcomplicating things. What's wrong with the following pattern?

```js
reactStamp(React).compose(
  container(PureFunc),
  mixin
);
```
http://codepen.io/troutowicz/pen/jbweRb?editors=001

**Comment by ericelliott** [comment]
:+1:

**Pull request closed without merging**